util/travis_utils.sh

```diff
@@ -11,7 +11,7 @@ TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE:-HEAD~1..HEAD}"
 
 # Extra variables
 LOCAL_BRANCH=$(git rev-parse --abbrev-ref HEAD)
-QMK_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE})
+QMK_CHANGES=$(git diff --name-only --diff-filter=d -n 1 ${TRAVIS_COMMIT_RANGE})
```

Comment:
This should actually be reverted, as it's not relevant to the purpose of your pull request.

[Contributing to QMK: General Guidelines - QMK Firmware Docs](https://docs.qmk.fm/#/contributing?id=general-guidelines):

> Separate PRs into logical units. For example, do not submit one PR covering two separate features, instead submit a separate PR for each feature.

```suggestion
QMK_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE})
```

Reply:
Done - opened #9963 to make this change. I assume merging this PR should wait for that to be merged first, since it [breaks CI](https://travis-ci.org/github/qmk/qmk_firmware/builds/715843818) otherwise.

Reply:
This doesn't necessarily have to wait for 9963; we're aware the CI tests get weird if files get moved or renamed. If you would like to wait for that PR to be merged though, we could do that.

Reply:
Cool, I wasn't aware of that - I certainly don't have any reason to wait to merge this PR then in that case.